docs/usage.rst

```diff
@@ -204,6 +204,27 @@ Close the software with ease of mind. Next time you open your software, all your
 you left it! KML Overlay supports **Saving Open files** so that you can jump back in, anytime!
 
 
+Multiple Flightpath Dockwidget
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+The topview has a dockwidget allowing to plot multiple flighttracks/operations on top of map.
+
+New flightpaths can be added or removed without crashing, and a clear visualization on map, with
+relevant geometries and styles.
+
```

Comment:
"without crashing" sounds strange here. I guess you don't want that users read that? remove it please.